src/components/DownloadButton.tsx

```diff
@@ -6,13 +6,23 @@ import { useTranslation } from 'react-i18next';
 interface DownloadButtonProps {
   contentURL: string;
   contentSize: string | undefined;
+  fileName: string;
 }
 
-const DownloadButton: FC<DownloadButtonProps> = ({ contentURL, contentSize }) => {
+const DownloadButton: FC<DownloadButtonProps> = ({ contentURL, contentSize, fileName }) => {
   const { t } = useTranslation();
+
   return (
-    <Button href={contentURL} size="large" startIcon={<CloudDownloadIcon />} variant="contained" color="primary">
-      {t('common.download')} {contentSize && ` (${contentSize})`}
+    <Button
+      href={contentURL}
+      download={fileName}
+      target="_blank"
+      rel="noopener norefferer"
+      size="large"
+      startIcon={<CloudDownloadIcon />}
+      variant="contained"
+      color="primary">
+      {t('resource.preview.open_in_new_tag')} {contentSize && ` (${contentSize})`}
     </Button>
```

Comment:
Syntes knappene så så mye bedre ut enn lenketekst, så tenkte at å få åpnet innholdet i ny fane og ny forklaring på hva som skjer er innafor?

Reply:
Det blir vel ikke helt riktig læll. Et .docx-dokument f.ex blir lagt rett til nedlasting på min browser iallefall

Reply:
Hva med "last ned / åpne i ny fane" ?

Reply:
Skulle jeg også lagt inn en task på NTH på ressursvisning? Eget backend endepunkt for kun nedlastning? Ser at digital postkasse til Posten har faktisk to-endepunkter, ett for å vise frem dokumenter og et for å laste de ned.

Reply:
smart.